Private/Format-Result.ps1

```diff
@@ -33,15 +33,23 @@ function Format-Result {
     $IssueTable = @{
```

Comment:
Create as an ordered dictionary to...keep these in order. 😊

Reply:
Order doesn't really matter on this.